rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.highlight/20_fvh.yml

```diff
@@ -22,7 +22,6 @@ setup:
   - do:
       index:
         index: test
-        refresh: true
```

Comment:
Why do we need to replace these with a refresh call? (I thought it should work as is).

Reply:
The test should indeed work as it is. I changed this because Index requests with immediate refresh should be used to make the doc searchable immediately after it is indexed, which is not useful here. But your comment makes me notice that the Refresh call I added was on all indices instead of just `test` (I pushed 38ff9b79a1e51ecfcd0d62f60a2ad943ba8514be)